engine/src/flutter/impeller/renderer/backend/vulkan/pipeline_vk.cc

```diff
@@ -376,8 +377,26 @@ fml::StatusOr<vk::UniquePipeline> MakePipeline(
     attachment_blend_state.push_back(
         ToVKPipelineColorBlendAttachmentState(color_desc.second));
   }
-  vk::PipelineColorBlendStateCreateInfo blend_state;
+
+  vk::StructureChain<vk::PipelineColorBlendStateCreateInfo,
+                     vk::PipelineColorBlendAdvancedStateCreateInfoEXT>
+      combined_blend_state;
+
+  auto& blend_state =
+      combined_blend_state.get<vk::PipelineColorBlendStateCreateInfo>();
   blend_state.setAttachments(attachment_blend_state);
+
+  if (caps->SupportsAdvancedBlendOperations()) {
```

Comment:
Can we have a unit test for this?  I'm thinking of mocking out SupportsAdvancedBlendOperations() and checking the calls to vulkan.